packages/terra-scroll/src/Scroll.jsx

```diff
@@ -21,7 +21,8 @@ const Scroll = ({
   refCallback,
   ...customProps
 }) => (
-  <div {...customProps} className={cx(['scroll', customProps.className])} ref={refCallback}>
+  // eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex
+  <div {...customProps} tabIndex="0" className={cx(['scroll', customProps.className])} ref={refCallback}>
```

Comment:
TabIndex needs to be selectively added. ContentContainers are used in nested fashion and we don't want all containers to supply a tabIndex. This also affects how an app and main elements are structured. The behavior will need to be evaluated when we the accessibility pass is implemented.

Reply:
@saket2403 Adding tabIndex here is causing the these [button-group](https://github.com/cerner/terra-core/blob/master/packages/terra-button-group/tests/wdio/button-group-spec.js#L41-L67) tests to fail because the first tab would put focus on the terra-scroll and the second tab would then put focus on the first button in the button group.

Reply:
Changed Implementation in 0823ae5
cc @benbcai  @dkasper-was-taken